docs/pages/guides/building-for-tv.mdx

```diff
@@ -0,0 +1,222 @@
+---
+title: Building for TV
+description: A guide for building an Expo app for an Apple TV or Android TV target.
```

Comment:
```suggestion
description: A guide for building an Expo app for an Android TV or Apple TV target.
```

Use the platform order as per our [writing style guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#referencing-android-ios-and-web).